tools/actions/commit-queue.sh

```diff
@@ -76,10 +80,18 @@ for pr in "$@"; do
     git node land --abort --yes
   else
     rm output
+    head=$(gitHubCurl "$(prUrl "$pr")" GET | jq '{ ref: .head.ref, remote: .head.repo.ssh_url, can_modify: .maintainer_can_modify }')
+    if [ "$(echo "$head" | jq -r .can_modify)" == "true" ]; then
+      git push --force "$(echo "$head" | jq -r .remote)" master:"$(echo "$head" | jq -r .ref)"
+      # Give GitHub time to sync before pushing to master
+      sleep 5s
```

Comment:
I might increase this to a whole minute or something, I just did this manually with a 15 seconds gap between push to PR branch and push to nodejs/node and I still got a race condition

Reply:
You have to wait until the origin/pull/123 ref has updated, which a git fetch can verify.